app/controllers/network_port_controller.rb

```diff
@@ -18,5 +19,15 @@ def self.display_methods
     %w(cloud_subnets floating_ips security_groups)
   end
 
+  def breadcrumbs_options
+    {
+      :breadcrumbs => [
+        {:title => _("Networks")},
+        {:title => _("Network Ports"), :url => "/#{controller_name}"},
```

Comment:
you should probably use `controller_url` consistently once it exists ;)

Reply:
yeah, it was written before the controller_url :smile_cat: